commands/CloudCommands.js

```diff
@@ -298,34 +298,29 @@ CloudCommand.prototype = extend(BaseCommand.prototype, {
 	},
 
 	compileCode: function (deviceType) {
-		if (!deviceType) {
-			console.error("\nPlease specify the target device type. eg. particle compile photon xxx\n");
-			return;
-		}
-
+		deviceType = deviceType.toLowerCase();
 		//defaults to 0 for core
 		var platform_id = 0;
 
 		switch (deviceType) {
-		  case "photon":
-		  case "p":
+			case "photon":
+			case "p":
 				deviceType = "photon";
 				platform_id = 6;
-		    break;
+				console.log("\nCompiling code for " + deviceType + "\n");
+				break;
 
-		  case "core":
+			case "core":
 			case "c":
 				deviceType = "core";
+				console.log("\nCompiling code for " + deviceType + "\n");
 				break;
 
-		  default:
-		    console.error("\nTarget device " + deviceType + " is not valid");
-				console.error("	eg. particle compile core xxx");
-				console.error("	eg. particle compile photon xxx\n");
-		    return;
+			default:
```

Comment:
Sure but this follows the behavior of the Web IDE where a "default" (the one you :star:) is compiled against.

Let me know how this could be changed ;)